src/js/stores/MesosStore.js

```diff
@@ -0,0 +1,221 @@
+var EventEmitter = require("events").EventEmitter;
+
+var AppDispatcher = require("../AppDispatcher");
+var InfoStore = require("../stores/InfoStore");
+var InfoActions = require("../actions/InfoActions");
+var InfoEvents = require("../events/InfoEvents");
+var MesosActions = require("../actions/MesosActions");
+var MesosEvents = require("../events/MesosEvents");
+
+const FILES_TTL = 60000;
+const STATE_TTL = 60000;
```

Comment:
Yep. We probably could cache the data even longer (where applicable) as we do invalidation not only on time but also if data is missing.